kf_lib_data_ingest/common/pandas_utils.py

```diff
@@ -274,7 +293,10 @@ def outer_merge(df1, df2, with_merge_detail_dfs=True, **kwargs):
     """
     kwargs['how'] = 'outer'
     kwargs['indicator'] = with_merge_detail_dfs
-    outer = merge_wo_duplicates(df1, df2, **kwargs)
+    outer = merge_wo_duplicates(df1, df2,
+                                left_name=kwargs.pop('left_name', None),
```

Comment:
A person calling outer_merge will never know about these left_name/right_name argument options because they aren't part of the function signature. Also why are you popping them here?